### ISSUE TITLE
Official Docker Image

### ISSUE THREAD

**Issue by mambax** [plantuml/plantuml#486]
I'd like to PR an official docker image published upon every release of yours with GitHub Actions. Also I might contribute a GHA for plantuml (an official) as I think PlantUML is amazing and deserves more attention and integration.

Would you be interested?

**Comment by arnaudroques** [comment]
Sure !
I must say that we have very little knowledge about docker and GitHub actions.
And unfortunately, we don't have time for this, since we focus on the core library.

So be creative :-) 
For example, you can add a /docker folder in the project if that can help, to better organize docker images.
Regarding GitHub Action, again, you can try some enhancements: you'll explain us when you will have finish.

Thanks again,

**Comment by mambax** [comment]
Cool! 🎉 

I have been invited to the server. Was that on purpose? Because I would rather need this repo here as I need to see its "releases". Not sure yet.

I will definitely try to add Docker and an action for it. Do you guys have an official Docker Repo?

**Comment by arnaudroques** [comment]
> I have been invited to the server. Was that on purpose? Because I would rather need this repo here as I need to see its "releases". Not sure yet.
> 

Yes, it was on purpose. I'm not sure that you would need this, but this way, we won't be a bottleneck.
We'll see if it's useful.

> I will definitely try to add Docker and an action for it. Do you guys have an official Docker Repo?

Yes, here https://hub.docker.com/r/plantuml/plantuml-server
Even if it's our own account, it sounds like black magic to us :-)

**Comment by mambax** [comment]
No worries I can lift some of the magic. Let me prepare my PR.
For the nature of the action, it does not belong in the server domain tbh. It belongs to the plantuml repo no? I am pretty sure because on the command line (and thus in GitHub Actions) we need the plantuml releases. But you do not need to "invite" me there, I can also fork it 🎉

**Comment by sepastian** [comment]
What is the status of this?

**Comment by gliptak** [comment]
https://github.com/plantuml/plantuml/pull/1067

later can be used during tag push GHA
